server/.vscode/launch.json

```diff
@@ -0,0 +1,102 @@
+{
```

Comment:
Could you remove this file, none of the main devs use vscode, and I don't really want any IDE specific stuff in the codebase.

Reply:
Best just add it to .gitignore instead.

Reply:
Good idea, will do.